RELEASE/scripts/autoscend/auto_settings.ash

```diff
@@ -34,12 +34,9 @@ boolean trackingSplitterFixer(string oldSetting, int day, string newSetting)
 
 void auto_settingsUpgrade()
 {
-	//upgrade settings from old format to new format
-	if(get_property("auto_debug") == "true")
-	{
-		set_property("auto_logLevel", "debug");
-	}
-
+	//upgrade settings from old format to new format.
+	//do not forget to add each old setting to auto_settingsDelete so it can be deleted after the upgrade is done.
```

Comment:
```suggestion
	//do not forget to add each old setting to auto_settingsDelete() so it can be deleted after the upgrade is done.
```
comment adjustment